cibuild.cmd

```diff
@@ -145,5 +146,11 @@ exit /b 1
 @REM Kill any instances of msbuild.exe to ensure that we never reuse nodes (e.g. if a non-roslyn CI run
 @REM left some floating around).
 
-taskkill /F /IM vbcscompiler.exe 2> nul
-taskkill /F /IM msbuild.exe 2> nul
+@REM An error-level of 1 means that the process was found, but could not be killed.
+taskkill /F /IM vbcscompiler.exe > nul
+if %ERRORLEVEL% == 1 exit /b 1
```

Comment:
I've added a message "killing all build related processes" 

I think this is generic enough, because we can call this TerminateBuildProcesses function on both a failure-path or just the normal "time to do a perf run" path.